chia/full_node/full_node.py

```diff
@@ -306,10 +314,9 @@ async def manage(self) -> AsyncIterator[None]:
             peak: Optional[BlockRecord] = self.blockchain.get_peak()
             if peak is None:
                 self.log.info(f"Initialized with empty blockchain time taken: {int(time_taken)}s")
-                num_unspent = await self.coin_store.num_unspent()
```

Comment:
The `.num_unspent` API is badly named and ill-conceived since a count could take a very long time depending on the specifics of the storage mechanism, and is not actually needed for any genuine purpose, so requiring it in the protocol applies a unnecessary constraint. We use a local function that more or less provides the same functionality. It doesn't catch all the same failure cases, but the prior method is just a mild sanity check anyway.